fastlane/Helpers/Test.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Test.swift
+//  FastlaneRunner
+//
+//  Created by Khanh on 01/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+enum Test {
+
+    static func buildAndTest(
+        environment: Constant.Environment,
+        targets: [String],
+        devices: [String]
+    ) {
+        scan(
+            scheme: .userDefined(environment.scheme),
+            devices: .userDefined(devices),
+            onlyTesting: targets,
+            codeCoverage: .userDefined(true),
+            outputDirectory: Constant.testOutputDirectoryPath,
+            xcodebuildFormatter: "xcpretty",
+            resultBundle: .userDefined(true),
```

Comment:
This is optional, I'd prefer to use `xcbeautify` than `xcpretty`. To use it you need to install it via `homebrew` first.
Fun fact: `xcbeautify` is written by [my younger brother](https://github.com/thii) 😄 (Its ownership is transferred to `tuist` now)

Reply:
> Similar to xcpretty, but faster.

That convinced me completely 😄. Wow [your brother](https://github.com/thii) has many cool repositories, I just followed ;).

Reply:
I updated it at https://github.com/nimblehq/nimble-crypto-ios/pull/23/commits/c51a3745552efbf4a4e4b0dec273e17cbb54fb3c :D.